src/Traits/FixturesTrait.php

```diff
@@ -188,8 +190,8 @@ protected function getTables(): array
 
     protected function exportContent($content, string $fixture): void
     {
-        if (env('FAIL_EXPORT_JSON', true)) {
-            $this->fail(preg_replace('/[ ]+/mu', ' ',
+        if (config('FAIL_EXPORT_JSON', true)) {
```

Comment:
let's implement package config file with this value

Reply:
@DenTray Reverted to the env